src/core/control/tools/ImageHandler.h

```diff
@@ -26,16 +26,42 @@ class ImageHandler {
     virtual ~ImageHandler();
 
 public:
-    bool insertImage(double x, double y);
-    // does the complete insertion based on create and add
-    bool insertImage(GFile* file, double x, double y);
-    // creates the image and does automatic scaling of the image
-    std::tuple<Image*, int, int> createImage(GFile* file, double x, double y);
-    bool addImageToDocument(Image* img, bool addUndoAction);
-    // scale down (only if necessary) the image so that the it fits on the page
-    // applies (potentially adjusted) width/height to the image
+    /**
+     * inserts an image scaled to the given size
+     */
+    auto insertImageWithSize(xoj::util::Rectangle<double> space) -> bool;
```

Comment:
The rectangles could be passed by ref
```suggestion
    auto insertImageWithSize(const xoj::util::Rectangle<double>& space) -> bool;
```